### PR TITLE
Fix PickerView that accept one-dimensional array as data property

### LINKED ISSUE #2775: PickerView's data prop does not support one-dimension array
- [x] I have searched the [issues](https://github.com/ant-design/ant-design-mobile/issues) of this repository and believe that this is not a duplicate.

### Version
2.2.3

### Environment
Mac OS 10.13.6, Chrome 68.0.3440.106

### Reproduction link
[https://codepen.io/anon/pen/bjJjxp](https://codepen.io/anon/pen/bjJjxp)

### Steps to reproduce
1. Copy the demo code of  PickerView on official doc site, then modify the `seasons` to one dimension array
   
     ```
    [
        { label: '春',value: '春'},
        { label: '夏',value: '夏'}
    ]

2. view the demo

### What is expected?
should display an picker view just like  demo

### What is actually happening?
nothing display, error on console: 
PickerView.js:40 Uncaught TypeError: col.map is not a function
    at PickerView.js:40
    at Array.map (<anonymous>)
    at PickerView._this.getCol (PickerView.js:36)
    at PickerView.render (PickerView.js:66)
![image](https://user-images.githubusercontent.com/3773928/44098115-e7d55d12-a011-11e8-8c4d-878c21f4c8d2.png)



---
Under official doc's description, `PickerView`'s `data` prop can be `Array<{value, label}> / Array<Array<{value, label}>>`，so  `[{},{}]` should be  effective as data source.


<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

### PR DESCRIPTION
**Pull request by oMaten**
Fixes #2775 

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [x] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/2815)

### CODE REVIEW

File: components/picker-view/PickerView.tsx [hunk 1]
```diff
@@ -32,9 +32,16 @@ export interface IPickerView {
 export default class PickerView extends React.Component<IPickerView, any> {
   static defaultProps = getDefaultProps();
 
+  isMultiPicker = () => {
+    if (!this.props.data) { return false; }
+    return Array.isArray(this.props.data[0]);
```
Comment (ziluo) [inline components/picker-view/PickerView.tsx]: 这里是否要考虑下isArra在移动端的兼容性？

Comment (oMaten) [reply components/picker-view/PickerView.tsx]: https://github.com/ant-design/ant-design-mobile/blob/master/components/action-sheet/index.tsx#L162 这里用了所以我就沿用了，http://kangax.github.io/compat-table/es5/#test-Array.isArray 看下这个兼容性是否满足？

**Comment by ziluo** [comment]
最好补一下测试用例 ~~

**Comment by oMaten** [comment]
已补充快照以及相关测试用例

**Pull request merged**